.travis.yml

```
@@ -81,13 +83,13 @@ jobs:
       script:
       - "ci/versioning/add_tag_to_git.sh"
       - "ci/versioning/set_next_version.sh"
-#     - stage: deploy
-#       cache: false
-#       if: branch = dev and type != pull_request
-#       install:
-#         - source charts/.travis/install.sh
-#       script:
-#       - travis_wait ci/helm/deploy.sh $DEPLOY_TARGET -f projects/saturn/views.yaml
+     - stage: deploy
+       cache: false
+       if: $SHOULD_DEPLOY and branch = dev and type != pull_request
```

You probably need the function `env` here (see https://docs.travis-ci.com/user/conditions-v1):
```suggestion
       if: env(SHOULD_DEPLOY) = true and branch = dev and type != pull_request
```
Although I remember that there were issues with using environment variables in conditions in Travis scripts.